packages/google_sign_in/ios/Classes/GoogleSignInPlugin.m

```diff
@@ -96,7 +96,12 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
     result(@([[GIDSignIn sharedInstance] hasAuthInKeychain]));
   } else if ([call.method isEqualToString:@"signIn"]) {
     if ([self setAccountRequest:result]) {
-      [[GIDSignIn sharedInstance] signIn];
+      @try {
+        [[GIDSignIn sharedInstance] signIn];
+      } @catch (NSException *e) {
+        result([FlutterError errorWithCode:@"google_sign_in" message:e.reason details:e.name]);
+        [e raise];
```

Comment:
debating myself if we should fail more graciously that was requested by the customer in https://github.com/flutter/flutter/issues/12734 or we should still crashing it to just have better error message to force user to fix it.

Reply:
Missing configuration is fatal. An app like this should never be released and there's no gracious failure for it. Raising FlutterError is the right way to go for this.